packages/boxel/index.js

```diff
@@ -81,7 +81,7 @@ module.exports = {
     // grab all js files from the addon directory, except utils
     let addonFiles = stew.find(
       this.treeGenerator(path.join(__dirname, 'addon')),
-      '**/*.+(js|ts|hbs)'
+      '**/*.+(js|ts|gts|hbs)'
```

Comment:
You were getting trolled by this clever code that builds the app reexports tree automatically. It didn't know about gts file extensions.